soc/arm/st_stm32/stm32f4/dts_fixup.h

```diff
@@ -412,4 +412,34 @@
 #define DT_ADC_1_CLOCK_BITS		DT_ST_STM32_ADC_40012000_CLOCK_BITS_0
 #define DT_ADC_1_CLOCK_BUS		DT_ST_STM32_ADC_40012000_CLOCK_BUS_0
 
+#define DT_CAN_1_BASE_ADDRESS           DT_ST_STM32_CAN_40006400_BASE_ADDRESS
+#define DT_CAN_1_BUS_SPEED                      DT_ST_STM32_CAN_40006400_BUS_SPEED
```

Comment:
Please get all of these aligned.

Reply:
@erwango  PS: you know all dst_fixup files seem to have strange alignment

Should I reformat them all (in a separate PR)?

Reply:
> @erwango PS: you know all dst_fixup files seem to have strange alignment

@lowlander indeed, but let's control we introduce aligned stuff.

> Should I reformat them all (in a separate PR)?

Sure, go ahead.